kafka_consumer_freshness_tracker/src/test/java/com/tesla/data/consumer/freshness/ConsumerFreshnessTest.java

```diff
@@ -374,16 +371,28 @@ public void testStrictModeConfigurationIsInvalidMissingServer() throws Exception
 
     ConsumerFreshness freshness = new ConsumerFreshness();
     freshness.burrow = burrow;
-    freshness.strict = true;
 
     String expected = String.format(
-            "strict mode on and the set of bootstrap servers in config is not the same as the " +
+            "the set of bootstrap servers in config is not the same as the " +
                     "set advertised by Burrow\nconfig: %s\nburrow: %s",
             String.join(", ", confBootstrapServers),
             String.join(", ", burrowBootstrapServers)
     );
 
+    // normal mode
+    Assert.assertEquals(Optional.of(expected), freshness.validateClusterConf(conf));
+
+    // strict mode
+    freshness.strict = true;
     Assert.assertEquals(Optional.of(expected), freshness.validateClusterConf(conf));
+
+    // RuntimeException should be thrown when attempting to setup Tracker with invalid configuration in strict mode
+    thrown.expect(RuntimeException.class);
```

Comment:
this should go just before the error is thrown, in this case, right before the `setupWithBurrow` call